lib/ExpensiMark.js

```diff
@@ -476,9 +470,28 @@ export default class ExpensiMark {
             }
             replacedText = replacedText.concat(textToCheck.substr(startIndex, (match.index - startIndex)));
 
+            // we want to avoid matching email address domains
+            let abort = false;
+            let shouldRetryByAtSign = false;
```

Comment:
What does "by at sign" mean? are we doing another try by removing the @ sign, by adding the @ sign, or by testing against a new regex with/out @ sign?

Reply:
I am also feeling that variable name is not reasonable.

I think `isReparsedByAutolink` is more readable

Reply:
How do you think about new one?